installer/Migration/V5_1_0/permission/api.yaml

```
@@ -322,6 +322,23 @@ apiv2_recruitment_candidate_shortlist:
     - { role: Admin, permission: { read: false, create: false, update: true, delete: false, self: false } }
     - { role: HiringManager, permission: { read: false, create: false, update: true, delete: false, self: false } }
 
+
+##########################
+#        I18N
+##########################
+
+apiv2_admin_localization_languages:
+  description: 'API-v2 Admin - Localization Languages'
+  api: OrangeHRM\Admin\Api\I18NLanguageAPI
+  module: admin
+  allowed:
+    read: true
+    create: false
+    update: true
+    delete: false
+  permissions:
+    - { role: Admin, permission: { read: true, create: false, update: true, delete: false, self: false } }
```

```suggestion
  permissions:
    - { role: Admin, permission: { read: true, create: false, update: true, delete: false } }
```
@SandeepaOHRM remove from recruitment APIs also